### ISSUE TITLE
Develop DOI Service User Guide and Training

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/doi-ui#184]
## 💡 Description

- [ ] write PDS DN user guide @jordanpadams
- [ ] prepare a 'mandatory training' for DN users @jordanpadams 
- [ ] communicate username/password and pds-gamma service to DN users for tests/training @jordanpadams 
- [ ] present / get feedback from users at a DOI WG @jordanpadams

**Comment by jordanpadams** [comment]
📆 May status: delayed until end of build due to A&A documentation tasks. No impact on build.

**Comment by jordanpadams** [comment]
📆 June status: Start delayed 1 sprint to support other documentation work. Low priority task not required for B14.0 delivery.

**Comment by jordanpadams** [comment]
📆 July status: In work. On schedule.

**Comment by jordanpadams** [comment]
📆 August status: Deferring due to lack of time to work on this. No impact on delivery to I&T. Will be worked in parallel with I&T and will be needed for delivery in December.

**Comment by jordanpadams** [comment]
📆 02/2024 status: Deferring due to other priorities. Blocked by bugs in DOI Editor, so this information is not needed at this time.
